packages/portfolio-deploy/src/portfolio-start.core.js

```diff
@@ -110,12 +111,17 @@ export const startPortfolio = async (permitted, configStruct) => {
 
   await permitted.consume.chainInfoPublished;
 
-  const { issuer } = permitted;
-  const [BLD, USDC, PoC26] = await Promise.all([
-    issuer.consume.BLD,
-    issuer.consume.USDC,
-    issuer.consume.PoC26,
-  ]);
+  const {
+    issuer,
+    consume: { agoricNames },
+  } = permitted;
+
+  const PoC26 = await issuer.consume.PoC26;
+  trace('startPortfolio: settled PoC26');
+  const USDC = await E(agoricNames).lookup('issuer', 'USDC');
+  trace('startPortfolio: settled USDC');
```

Comment:
@amessbee I'm curious what prompted this change. It turns out to be a very important one.